Packs/ApiModules/Scripts/CSVFeedApiModule/CSVFeedApiModule.py

```diff
@@ -142,6 +142,14 @@ def build_iterator(self, **kwargs):
 
             try:
                 r = _session.send(prepreq, **kwargs)
+            except requests.exceptions.SSLError as exception:
+                err_msg = 'SSL Certificate Verification Failed - try selecting \'Trust any certificate\' checkbox in' \
+                          ' the integration configuration.'
+                raise DemistoException(err_msg, exception)
+            except requests.exceptions.ProxyError as exception:
+                err_msg = 'Proxy Error - if the \'Use system proxy\' checkbox in the integration configuration is' \
+                          ' selected, try clearing the checkbox.'
+                raise DemistoException(err_msg, exception)
             except requests.ConnectionError:
                 raise requests.ConnectionError('Failed to establish a new connection.'
```

Comment:
Let's make sure we keep the original message. By raising a new error we lose the original cause for the connection error. It could be various reasons (dns lookup, timeout).

Reply:
updated the exception handling according to the `BaseClient.http_request` function